.editorconfig

```diff
@@ -0,0 +1,80 @@
+root = true
+
+## Repository Configurations
+############################
+[.{git*,editorconfig,*.yml}]
```

Comment:
The file extension `yml` does not work here because it expands to `.*.yml`.

This works: `[{.git*,.editorconfig,*.yml}]`

Tested with Sublime Text.

Reply:
That's intentional, e.g. `.travis.yml` and other repo configs (Circle CI, etc.).

Reply:
Ok